Urania/Urania.Desktop/MainViewModel.cs

```diff
@@ -36,6 +52,34 @@ public class MainViewModel : INotifyPropertyChanged {
         protected void OnPropertyChanged([CallerMemberName] string name = null) {
             PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(name));
         }
-        
+        private void CheckCalculatePossibility() {
+            if (WireParameters.IsValid == true) {
+                int notNullCount = 0;
+                if (WireParameters.Id != null) notNullCount++;
+                if (WireParameters.Od != null) notNullCount++;
+                if (WireParameters.Wd != null) notNullCount++;
+                if (WireParameters.Ar != null) notNullCount++;
+
+                
```

Comment:
nie lepiej tutaj zmienić IsParametersCountAboveTwo = false; zamiast lijek 66 i 71, a pozostawić 76 jak jest?

Reply:
w ogóle można to wyzucić na początek metody przed if to też odpadnie nam to z liniki 80

Reply:
w sumie to samo z CanCalculate - przed pierwszym ifem przypisz false, a zmieniaj tylko na true